docs/pubsub-api.rst

```diff
@@ -0,0 +1,255 @@
+``gcloud.pubsub`` API
+=====================
+
+Connection / Authorization
+--------------------------
+
+- Inferred defaults used to create connection if none configured explicitly:
```

Comment:
Because the story is going to be pretty much identical to the `datastore` / `storage` cases, I didn't include it in this sketch.